### ISSUE TITLE
How to Clear History and Cookies After the Browser Closes?

### ISSUE THREAD

**Issue by amarakon** [fanglingsu/vimb#702]
I want vimb to clear all history, logins, cookies and stuff like that when all vimb windows have been closed.

**Comment by amarakon** [comment]
Yep it works. But the only thing it didn't clear was the sites I've visited, and I want it to clear that too.

**Comment by fanglingsu** [comment]
@amarakon You mean the entries added in `.local/share/vimb/history`. You could delete this file by yourself. And to be save you should also delete the `.local/share/vimb/closed` which stores the last `closed-max-items` closed urls. Or you could `:set  closed-max-items=0` so that there is no url written to the file when vimb is closed.
